<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>884</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="choseBtn2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>280</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 73, 153);
color: rgb(255, 255, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>chose image 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="choseBtn1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 73, 153);
color: rgb(255, 255, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>chose image 1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="choseBtn3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>360</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 73, 153);
color: rgb(255, 255, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>chose image 3</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="showMapBtn">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>720</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 73, 153);
color: rgb(255, 255, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>show map</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 73, 153);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;SSI MAPS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="logoo.qrc"/>
 </resources>
 <connections/>
</ui>
